draft-ietf-quic-transport.md

```diff
@@ -5495,6 +5497,16 @@ Reason Phrase:
   This SHOULD be a UTF-8 encoded string {{!RFC3629}}.
 
 
+## HANDSHAKE_DONE frame {#handshake-done-frame}
+
+The server uses the HANDSHAKE_DONE frame (type=0x1e) to signal confirmation of
+the handshake to the client.  The HANDSHAKE_DONE frame contains no additional
+fields.
+
+This frame can only be sent by the server. A server MUST treat receipt of a
+HANDSHAKE_DONE frame as a connection error of type PROTOCOL_VIOLATION.
+
+
```

Comment:
Can we add the following paragraph to this section:

```
Clients MAY treat receipt of an acknowledgment for a packet it sent with 1-RTT
keys as receipt of a HANDSHAKE_DONE frame.  This can be implemented by
recording the lowest packet number sent with 1-RTT keys, and the highest
value of the Largest Acknowledged field in any received 1-RTT ACK frame:
once the latter is higher than or equal to the former, the handshake can be
confirmed.
```

Reply:
s / the handshake can be confirmed / the handshake is confirmed

Reply:
I intentionally made it "can be" because this is a MAY

Reply:
Marking this branch resolved because this text has been added.